Header header

#Напряжение батареи [B]
float64 battery_voltage

#Ток батареи [A]
float64 battery_current